REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,131 @@
+# Repository Guidelines
+
+This document describes the scope, status, and lifecycle of repositories within [Cilium](https://github.com/cilium).
+
+You can find the list of all repositories, along with their scope and status, in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md).
+
+**Table of Contents**
+
+* [Scope](#scope)
+  * [```Core```](#core)
+  * [```Ecosystem```](#ecosystem)
+  * [```Infra```](#infra)
+  * [```Special```](#special)
+* [Status](#status)
+  * [```STABLE```](#stable)
+  * [```INCUBATING```](#incubating)
+  * [```SANDBOX```](#sandbox)
+  * [```DEPRECATED```](#deprecated)
+* [Lifecycle](#lifecycle)
+  * [Addition](#addition)
+    * [Adding ```Core``` repositories](#adding-core-repositories)
+  * [Change of scope or status](#change-of-scope-or-status)
+    * [Archival](#archival)
+* [Credit](#credit)
+
+## Scope
+
+Each repository within the Cilium GitHub organization is assigned one of four scopes: ```Core```, ```Ecosystem```, ```Infra```, or ```Special```. This assigned scope helps to convey each repository's purpose and responsibility in the overall functioning of the project.
+
+### ```Core```
```

Comment:
maybe rather than "core" we should name it something to reflect that these are the end user facing projects/repos